data-reporting-service/post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -35,39 +33,19 @@ public class PostProcessingService {
     private final ProviderRepository providerRepository;
     private final OrganizationRepository organizationRepository;
     private final InvestigationRepository investigationRepository;
+    private final NotificationRepository notificationRepository;
 
-    @KafkaListener(topics = {"${spring.kafka.topic.investigation}"})
-    public void postProcessInvestigationMessage(@Header(KafkaHeaders.RECEIVED_KEY) String key,
-                                                @Header(KafkaHeaders.RECEIVED_TOPIC) String topic) {
+    @KafkaListener(topics = {
```

Comment:
do we have to add the dead letter queue implementation here. If it will be taken as a separate work, we can create a follow up ticket.